src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -2445,7 +2445,20 @@ static NativeLibrary loadLibrary(Class<?> fromClass, String name) {
     /*
      * Invoked in the VM class linking code.
      */
-    static long findNative(ClassLoader loader, String entryName) {
+    static long findNative(ClassLoader loader, Class<?> clazz, String entryName, String javaName) {
```

Comment:
I think this is another place where `@param` descriptions would help as it's not immediately clear that "javaName" is a method name.